.vscode/scripts/runtime/k3d/deploy_runtime.sh

```diff
@@ -16,25 +16,39 @@
 if ! helm status vehicleappruntime &> /dev/null
 then
     ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../../../.." )
+    DEPENDENCIES=$(cat $ROOT_DIRECTORY/AppManifest.json | jq .[].dependencies)
+    SERVICES=$(echo $DEPENDENCIES | jq '.services')
 
-    VEHICLEDATABROKER_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databrokerimage.version | tr -d '"')
-    SEATSERVICE_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.version | tr -d '"')
-    FEEDERCAN_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .feedercan.version | tr -d '"')
+    # Get Data from AppManifest.json and save to ENV
+    UTILS_DIRECTORY=$(dirname `cd ..; dirname "$0"`)/utils
```

Comment:
As discussed, the Integration test is failing due to the relative path of the UTILS_DIRECTORY  directory, Please use the ROOT_DIRECTORY for all scripts for consistency. 

Thanks!